objectivec/Tests/CocoaPods/run_tests.sh

```diff
@@ -125,6 +125,10 @@ do_test() {
   # fail on Travis since those machines don't have a Code Signing identity.
   if  [[ "${TEST_NAME}" == iOS* ]] ; then
     xcodebuild_args+=( "-sdk" "iphonesimulator" "ONLY_ACTIVE_ARCH=NO" )
+    # Apparently the destination flag is required to avoid "Unsupported architecture"
+    # errors.
+    xcodebuild_args+=( "-destination" )
+    xcodebuild_args+=( "platform=iOS Simulator,name=iPad 2,OS=9.3" )
```

Comment:
I'd likely format this like:

```shell
xcodebuild_args+=(
  -destination "platform=iOS Simulator,name=iPad 2,OS=9.3"
)
```

Since it shows how the args go together.

Yea, hardcoding versions sorta sucks.  In the `full_mac_build` script, I tried to deal with a few versions:
https://github.com/google/protobuf/blob/master/objectivec/DevTools/full_mac_build.sh#L229

I'm not sure that's needed here since I don't expect as many folks to try running these tests.